modules.d/01systemd-modules-load/module-setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+
+    # If the binary(s) requirements are not fulfilled
+    # return 1 to not include the binary.
+    require_binaries $systemdutildir/systemd-modules-load || return 1
+
+    # If the module dependency requirements are not fulfilled
+    # return 1 to not include the required module(s).
+    if ! dracut_module_included "systemd"; then
+        derror "systemd-modules-load needs systemd in the initramfs."
+        return 1
+    fi
+
+    # Return 255 to only include the module, if another module requires it.
+    return 255
+
+}
+
+# Module dependency requirements.
+depends() {
+
+    # This module has external dependency on the systemd module.
+    echo systemd
+    # Return 0 to include the dependent systemd module in the initramfs.
+    return 0
+
+}
+
+# Install the required file(s) for the module in the initramfs.
+install() {
+
+        # Create systemd-modules-load related directories.
+        inst_dir    $modulesload
+        inst_dir    $modulesloadconfdir
+
+        # Install related files for systemd-modules-load
+        inst_multiple -o \
+        # Install the systemd type service unit for systemd-modules-load.
+        $systemdsystemunitdir/systemd-modules-load.service \
+        # Install the binary executable(s) for systemd-modules-load.
+        $systemdutildir/systemd-modules-load
```

Comment:
Line continuation cannot be broken by comments

Reply:
Fixed along with removing the module check(s)